steering-files/src/main/resources/org/hps/steering/readout/Future2pt3TrigPulse_noAddNoise.lcsim

```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lcsim xmlns:xs="http://www.w3.org/2001/XMLSchema-instance"
+       xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/lcsim/1.0/lcsim.xsd">
+    <!-- 
+      @Readout steering file with pairs trigger for 2019 MC.

```

Comment:
Is this the correct comment? 2019 had 4.55 GeV beam?

Reply:
Good catch. Comments have been corrected.